conf/minifi.properties

```diff
@@ -51,7 +51,7 @@ nifi.database.content.repository.directory.default=${MINIFI_HOME}/content_reposi
 #nifi.c2.flow.base.url=
 #nifi.c2.rest.url=
 #nifi.c2.rest.url.ack=
-nifi.c2.root.classes=DeviceInfoNode,AgentInformation,FlowInformation
+nifi.c2.root.classes=DeviceInfoNode,AgentInformationWithoutManifest,FlowInformation
```

Comment:
I think changing this by default could cause problems.
If someone wants to use an older C2 server with newer agents without adapting the C2 server to send DESCRIBE manifest requests when needed would have their workflow broken.
I think the possibility of changing to lightweight heartbeats should be documented in `C2.md`, and even a commented out line here with a `nifi.c2.root.classes` containing `AgentInformationWithoutManifest` (and an explanation of what it does) would be fine, but changing it by default does not seem safe.

Reply:
Ok